recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/doma.bb

```diff
@@ -23,8 +21,6 @@ DOMA_CONFIG_salvator-x-m3-xt = "doma-salvator-x-m3.cfg"
 DOMA_CONFIG_salvator-x-h3-xt = "doma-salvator-x-h3.cfg"
 DOMA_CONFIG_salvator-xs-h3-xt = "doma-salvator-xs-h3.cfg"
 DOMA_CONFIG_salvator-x-h3-4x2g-xt = "doma-salvator-x-h3-4x2g.cfg"
-DOMA_CONFIG_m3ulcb-xt = "doma-m3ulcb.cfg"
```

Comment:
please do not add doma first and then remove them

Reply:
no problem, will squash https://github.com/xen-troops/meta-xt-prod-gen3-test/pull/69/commits/a4e05108d6a372fd08f8526eeb82b3d1dd1634b4 and https://github.com/xen-troops/meta-xt-prod-gen3-test/pull/69/commits/ad13329a1956bdbec1311c802e3a53021ee93755